### ISSUE TITLE
Bad UX: Scanning Second device qr code with normal qr code scanner while already logged in 

### ISSUE THREAD

**Issue by Simon-Laux** [deltachat/deltachat-desktop#3569]
Funny UX when adding second device:

1. I'm starting DC Desktop with a fresh database
2. Clicking "Add as second device"
3. Scan QR code from DC Mobile
4. Desktop says "Login first"
5. I log in with my mail account
6. Repeat the steps, now it says "Logout first" hahaha
7. Couldn't find how to "log out", searching for a while ..
8. I went into the switch-account menu, adding second device from there
9. It works!

(originally reported by @adzialocha in dev chat)

Solution:
1. it should ask if you want to import account
2. and then switch accounts to a new one (new context)
3. and start the process of importing

Basically replicate flow of scanning `DCACCOUNT:` qr code while already logged in to another account (a configured context)

**Comment by Simon-Laux** [comment]
seems to be solved on master
